src/App.js

```diff
@@ -1,16 +1,44 @@
 import React from 'react';
 import './App.css';
 import chatMessages from './data/messages.json';
+import ChatLog from './components/ChatLog';
+import { useState } from 'react';
 
 const App = () => {
+  const [messageData, setMessageData] = useState(chatMessages);
+
+  const updateMessageData = (updatedData) => {
+    const updatedMessageData = messageData.map((message) => {
+      if (message.id === updatedData.id) {
+        return updatedData;
+      } else {
+        return message;
+      }
+    });
+
+    setMessageData(updatedMessageData);
+  };
+
+  const countLikes = (messages) => {
+    let likes = 0;
+    messages.forEach((message) => {
+      if (message.liked) {
+        likes++;
+      }
+    });
+    return likes;
+  };
+
+  const likes = countLikes(messageData);
+
   return (
     <div id="App">
       <header>
-        <h1>Application title</h1>
+        <h1>Chat Log</h1>
+        <section>{likes} ❤️s</section>
```

Comment:
This is a bit of personal preference, but since we only use the variable `likes` in 1 place, we could directly call the function here rather than storing the value in `likes`:
```suggestion
        <section>{countLikes(messageData)} ❤️s</section>
```